tango_ros_common/tango_ros_native/src/tango_ros_node.cpp

```diff
@@ -419,6 +410,21 @@ TangoErrorType TangoRosNode::OnTangoServiceConnected() {
   } else {
     occupancy_grid_publisher_.shutdown();
   }
+  if (fisheye_camera_available_) {
+    try {
+        fisheye_camera_publisher_ =
+            image_transport_->advertiseCamera(FISHEYE_IMAGE_TOPIC_NAME,
+                                              queue_size, latching);
+        fisheye_rectified_image_publisher_ =
+            image_transport_->advertise(FISHEYE_RECTIFIED_IMAGE_TOPIC_NAME,
+                                       queue_size, latching);
+      } catch (const image_transport::Exception& e) {
+        LOG(ERROR) << "Error while creating fisheye image transport publishers" << e.what();
```

Comment:
I see, in this case we could return an error if this exception occurs. What do you think?